.github/workflows/flake-updates-input.yml

```diff
@@ -1,10 +1,16 @@
-name: "Update flakes - nixpkgs-update"
+name: "Update flakes - input"
 on:
   repository_dispatch:
   workflow_dispatch:
+    inputs:
+      flakeInput:
+        description: "flake input to update"
+        required: true
+        type: string
```

Comment:
I didn't get where this is passed from.

Reply:
There would be another field for text input next to the branch drop down.

![input](https://github.com/nix-community/infra/assets/59103226/7ed8a5f3-5ed0-4f0a-9517-c20a49bb8925)